packages/lwt/lwt.2.5.1/opam

```diff
@@ -40,5 +40,6 @@ depopts: [
 conflicts: [
  "react" {<"1.0.0"}
  "ssl" {< "0.5.0"}
+ "camlp4" {= "4.02+7"} (* lacks "camlp4.quotations.o" *)
```

Comment:
I tried to compile lwt.2.5.1 + camlp4.4.02+7 and got a failure of ocamlfind requiring camlp4.quotations.o. I don't have the log anymore.

However, in 2.5.1 :
```
╰─➤ grep quotations.o **/* 2> /dev/null                                 [14:39]
myocamlbuild.ml:    "camlp4.quotations.o";
_oasis:  BuildDepends: camlp4, camlp4.quotations.o, camlp4.extend
_oasis:  BuildDepends: camlp4, camlp4.quotations.o
Binary file setup-dev.exe matches
setup.ml:                           FindlibPackage ("camlp4.quotations.o", None);
setup.ml:                           FindlibPackage ("camlp4.quotations.o", None)
_tags:<syntax/*.ml{,i,y}>: package(camlp4.quotations.o)
_tags:<examples/unix/logging.{native,byte}>: package(camlp4.quotations.o)
_tags:<examples/unix/relay.{native,byte}>: package(camlp4.quotations.o)
_tags:<examples/unix/parallelize.{native,byte}>: package(camlp4.quotations.o)
_tags:<examples/unix/*.ml{,i,y}>: package(camlp4.quotations.o)
```